examples/guided.py

```diff
@@ -1,20 +1,23 @@
-import getpass, time, json, os, logging
+import json
+import logging
+import time
+
 import archinstall
 from archinstall.lib.hardware import hasUEFI
-from archinstall.lib.profiles import Profile
 
 if archinstall.arguments.get('help'):
 	print("See `man archinstall` for help.")
 	exit(0)
 
 # For support reasons, we'll log the disk layout pre installation to match against post-installation layout
-archinstall.log(f"Disk states before installing: {archinstall.disk_layouts()}", level=archinstall.LOG_LEVELS.Debug)
+archinstall.log(f"Disk states before installing: {archinstall.disk_layouts()}", level=archinstall.LogLevels.Debug)
```

Comment:
Note to self, this should be `logging.DEBUG` not `LogLevels.Debug`.
It's an issue outside of this PR but caught it here. `LOG_LEVELS` is deprecated and going away.

Reply:
Resolved in latest commit